script.sql

```diff
@@ -1,147 +1,85 @@
-CREATE DATABASE Lab2;
-USE Lab2;
+-- CREATE
 CREATE TABLE Publisher ( 
-id INT,
-Name varchar(60),
- primary key (id)
+        id INT,
+        name VARCHAR(60),
+        PRIMARY KEY (id)
```

Comment:
I change reserved key words to UPPERCASE as a good practice. We could change it later if u like to use lowercase.